policy/release/trusted_task.rego

```diff
@@ -145,6 +147,36 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [])
 }
 
+# METADATA
+# title: Trusted parameters
+# description: >-
+#   Confirm certain parameters provided to each builder Task have come from trusted Tasks.
+# custom:
+#   short_name: trusted_parameters
```

Comment:
I considered an alternative to introducing a new policy rule. We could instead extend how the `_artifact_chain` is computed to take into account this logic. I was half-way through the implementation when I realized that this would exclude the check when Trusted Artifacts are not in use.  Maybe it's something we move towards in the future, but I do like the benefit of applying this rule in call cases.

Reply:
Yeah I think this is a generic constraint, we don't want parameters from untrusted tasks to go to any trusted task when not using trusted artifacts; or to any trusted tasks on the artifact chain when using trusted artifacts. Not sure if we need two variants of this rule for those two cases, perhaps...